app.js

```diff
@@ -1,33 +1,21 @@
 console.log("JS Loaded");
 
-
-
 $('#btn').click(function(){
  var value = $('#whichCrypto').val();
```

Comment:
Indentation is not consistent. Looks like one space, and it's not lined up with the code below it.